internal/tx_indexer/indexer.go

```diff
@@ -112,6 +112,16 @@ func (fi *FeeIndexer) updateTxStatus(ctx context.Context, tx storage.Tx) error {
 		if err != nil {
 			return fmt.Errorf("failed to insert fee: %w", err)
 		}
+	} else if tx.PluginID == types.PluginVultisigFees_feee {
```

Comment:
How about
```suggestion
	} 
	if tx.PluginID == types.PluginVultisigFees_feee {
```
?
Would be slightly easier to read, without modifying the funcionality